extensions/visual-studio-code-project-manager/src/search-project-manager-projects.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  Detail,
+  environment,
+  getPreferenceValues,
+  List,
+  OpenAction,
+  OpenWithAction,
+  ShowInFinderAction,
+  TrashAction,
+} from "@raycast/api";
+import { existsSync, readFileSync } from "fs";
+import { homedir } from "os";
+import { dirname } from "path";
+import { ReactElement } from "react";
+import tildify from "tildify";
+import { Preferences, ProjectEntry } from "./types";
+
+const STORAGE = `${homedir()}/Library/Application Support/Code/User/globalStorage/alefragnani.project-manager/projects.json`;
```

Comment:
@MarkusLanger what do we need to do to target vscode insiders instead?

Reply:
@ryan-efendy I created a pull request to support VSCode Insiders  https://github.com/raycast/extensions/pull/1207